Assets/_Scripts/BuildingSpawner.cs

```diff
@@ -15,6 +17,8 @@ public class BuildingSpawner : MonoBehaviour {
     bool usedOnce = false;
 	// Use this for initialization
 	void Start () {
+        amountSpawned = 0;
+        maxBuildings = 5;
```

Comment:
These assignments should be in the constructor or not exist at all - as they are they overwrite the values you provide in the editor. E.g 'public int maxBuildings= 5' would let you enter a number in the editor, but have a default of 5, which is what I think you're after.